tokio-fs/src/file/mod.rs

```diff
@@ -73,15 +130,45 @@ impl File {
     /// runtime or if the underlying [`create`] call results in an error.
     ///
     /// [`create`]: https://doc.rust-lang.org/std/fs/struct.File.html#method.create
+    ///
+    /// # Examples
+    ///
+    /// ```no_run
+    /// # extern crate tokio;
+    /// use tokio::prelude::*;
+    /// fn main() {
+    ///     let task = tokio::fs::File::create("foo.txt").and_then(|file| {
+    ///         // do something with the created file ...
+    ///         file.metadata().map(|md| println!("{:?}", md))
+    ///     }).map_err(|e| {
+    ///         //handle errors
+    ///         eprintln!("{}", e);
+    ///     });
+    ///     tokio::run(task);
+    /// }
+    /// ```
     pub fn create<P>(path: P) -> CreateFuture<P>
     where P: AsRef<Path> + Send + 'static,
     {
         CreateFuture::new(path)
     }
 
-    /// Convert a [`std::fs::File`][std] to a `tokio_fs::File`.
+    /// Convert a [`std::fs::File`][std] to a [`tokio_fs::File`][file].
     ///
     /// [std]: https://doc.rust-lang.org/std/fs/struct.File.html
+    /// [file]: struct.File.html
+    ///
+    /// Examples
+    /// ```no_run
+    /// # extern crate tokio;
+    /// use std::fs::File;
+    /// 
+    /// fn main() -> Result<(), Box<std::error::Error>> {
```

Comment:
I think drop the `Result` (and the corresponding `Ok`) from this.  We aren't going to run it in CI, so we can be a little more terse.

Reply:
cool. I have dropped the result and now just have a single unwrap.